### PR TITLE
adding a central axis example

### PR DESCRIPTION
**Pull request by ebrillhart**
This is in reference to #8 and creates a horizontal stacked bar chart with positive and negative values along a central axis. The data displays wonderfully and so do the percentage labels, but I'm having a difficult time with the axis - need to figure out how to make the axis labels 1) display appropriately on the appropriate data bar 2) show up in front of the data instead of behind it 3) center on the axis instead of be offset to one side. Ideally, the axis itself will remain behind the data while the labels come in front.

### CODE REVIEW

File: demo/tutorials/custom-central-axis.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            tickValues={[
+              "Smartphone",
+              "Laptop",
+              "Television",
+              "Tablet",
+              "Fitness Monitor",
+              "Smartwatch",
+              "Connected Surveillance Camera",
+              "Smart Thermostat",
+              "Personal Drones"
```
Comment (boygirl) [inline demo/tutorials/custom-central-axis.js]: I love your examples. Personal Drones.

Comment (ebrillhart) [inline demo/tutorials/custom-central-axis.js]: haha thanks but I can't take credit! it was in the example chart. super curious how they chose these specific devices. connected surveillance camera?

**Comment by ebrillhart** [comment]
@boygirl I think this is done if it looks good to you!

File: demo/tutorials/custom-central-axis.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+              ]}
+              labels={(data) => (`${Math.abs(data.y)}%`)}
+            />
+          </VictoryStack>
+
+          <VictoryAxis dependentAxis
+            style={{
+              axis: {strokeWidth: 1, stroke: "transparent"},
+              ticks: {stroke: "transparent"},
+              tickLabels: {WebkitTransform: "translateX(30px)", fontSize: 5, fill: "black"}
```
Comment (boygirl) [inline demo/tutorials/custom-central-axis.js]: please use the standard svg transform syntax `transform: "translate(30)"`

**Comment by boygirl** [comment]
@ebrillhart sorry to push to your branch. I made a couple simplifying changes and added some explanatory comments. The one with `tickLabelComponent` I _just_ enabled in `VictoryChart` a couple days ago. Pull master of this repo for updated deps.

**Comment by ebrillhart** [comment]
that's okay! the explanations make sense. can we talk about this a little more during our pairing session today?

**Comment by boygirl** [comment]
Of course!

**Comment by ebrillhart** [comment]
this looks great post functional npm install! shall we merge?

**Comment by boygirl** [comment]
Yep, merge away!

**Pull request merged**